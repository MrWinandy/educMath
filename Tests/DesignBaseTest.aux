\relax 
\providecommand\hyper@newdestlabel[2]{}
\providecommand\zref@newlabel[2]{}
\providecommand*\new@tpo@label[2]{}
\catcode `:\active 
\catcode `;\active 
\catcode `!\active 
\catcode `?\active 
\providecommand\HyperFirstAtBeginDocument{\AtBeginDocument}
\HyperFirstAtBeginDocument{\ifx\hyper@anchor\@undefined
\global\let\oldcontentsline\contentsline
\gdef\contentsline#1#2#3#4{\oldcontentsline{#1}{#2}{#3}}
\global\let\oldnewlabel\newlabel
\gdef\newlabel#1#2{\newlabelxx{#1}#2}
\gdef\newlabelxx#1#2#3#4#5#6{\oldnewlabel{#1}{{#2}{#3}}}
\AtEndDocument{\ifx\hyper@anchor\@undefined
\let\contentsline\oldcontentsline
\let\newlabel\oldnewlabel
\fi}
\fi}
\global\let\hyper@last\relax 
\gdef\HyperFirstAtBeginDocument#1{#1}
\providecommand\HyField@AuxAddToFields[1]{}
\providecommand\HyField@AuxAddToCoFields[2]{}
\providecommand\BKM@entry[2]{}
\BKM@entry{id=1,dest={636861707465722E31},srcline={6}}{666972737454657374}
\BKM@entry{id=2,dest={73656374696F6E2E312E31},srcline={8}}{53756254657374}
\babel@aux{french}{}
\babel@aux{french}{}
\@writefile{toc}{\contentsline {chapter}{\numberline {1}firstTest}{1}{chapter.1}\protected@file@percent }
\@writefile{lof}{\addvspace {10\p@ }}
\@writefile{lot}{\addvspace {10\p@ }}
\@writefile{toc}{\contentsline {section}{\numberline {1.1}SubTest}{1}{section.1.1}\protected@file@percent }
\pgfsyspdfmark {pgfid3}{3915828}{31602480}
