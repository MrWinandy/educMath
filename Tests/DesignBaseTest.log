This is pdfTeX, Version 3.14159265-2.6-1.40.17 (TeX Live 2016/W32TeX) (preloaded format=pdflatex 2016.9.26)  31 AUG 2020 12:05
entering extended mode
 restricted \write18 enabled.
 %&-line parsing enabled.
**./DesignBaseTest.tex
(./DesignBaseTest.tex
LaTeX2e <2016/03/31> patch level 3
Babel <3.9r> and hyphenation patterns for 83 language(s) loaded.
(d:/texlive/2016/texmf-dist/tex/latex/koma-script/scrbook.cls
Document Class: scrbook 2016/06/14 v3.21 KOMA-Script document class (book)
(d:/texlive/2016/texmf-dist/tex/latex/koma-script/scrkbase.sty
Package: scrkbase 2016/06/14 v3.21 KOMA-Script package (KOMA-Script-dependent b
asics and keyval usage)

(d:/texlive/2016/texmf-dist/tex/latex/koma-script/scrbase.sty
Package: scrbase 2016/06/14 v3.21 KOMA-Script package (KOMA-Script-independent 
basics and keyval usage)

(d:/texlive/2016/texmf-dist/tex/latex/graphics/keyval.sty
Package: keyval 2014/10/28 v1.15 key=value parser (DPC)
\KV@toks@=\toks14
)
(d:/texlive/2016/texmf-dist/tex/latex/koma-script/scrlfile.sty
Package: scrlfile 2016/06/14 v3.21 KOMA-Script package (loading files)

Package scrlfile, 2016/06/14 v3.21 KOMA-Script package (loading files)
                  Copyright (C) Markus Kohm

))) (d:/texlive/2016/texmf-dist/tex/latex/koma-script/tocbasic.sty
Package: tocbasic 2016/06/14 v3.21 KOMA-Script package (handling toc-files)
\scr@dte@tocline@numberwidth=\skip41
\scr@dte@tocline@numbox=\box26
)
Package tocbasic Info: omitting babel extension for `toc'
(tocbasic)             because of feature `nobabel' available
(tocbasic)             for `toc' on input line 132.
Package tocbasic Info: omitting babel extension for `lof'
(tocbasic)             because of feature `nobabel' available
(tocbasic)             for `lof' on input line 133.
Package tocbasic Info: omitting babel extension for `lot'
(tocbasic)             because of feature `nobabel' available
(tocbasic)             for `lot' on input line 134.
Package tocbasic Info: defining new hook before heading of `' on input line 160
2.
Package scrbook Info: You've used standard option `11pt'.
(scrbook)             This is correct!
(scrbook)             Internally I'm using `fontsize=11pt'.
(scrbook)             If you'd like to set the option with \KOMAoptions,
(scrbook)             you'd have to use `fontsize=11pt' there
(scrbook)             instead of `11pt', too.
Class scrbook Info: You've used standard option `oneside'.
(scrbook)           This is correct!
(scrbook)           Internally I'm using `twoside=false'.
(scrbook)           If you'd like to set the option with \KOMAoptions,
(scrbook)           you'd have to use `twoside=false' there
(scrbook)           instead of `oneside', too.
Class scrbook Info: File `scrsize11pt.clo' used to setup font sizes on input li
ne 2269.

(d:/texlive/2016/texmf-dist/tex/latex/koma-script/scrsize11pt.clo
File: scrsize11pt.clo 2016/06/14 v3.21 KOMA-Script font size class option (11pt
)
)
(d:/texlive/2016/texmf-dist/tex/latex/koma-script/typearea.sty
Package: typearea 2016/06/14 v3.21 KOMA-Script package (type area)

Package typearea, 2016/06/14 v3.21 KOMA-Script package (type area)
                  Copyright (C) Frank Neukam, 1992-1994
                  Copyright (C) Markus Kohm, 1994-

\ta@bcor=\skip42
\ta@div=\count79
Package typearea Info: You've used standard option `a4paper'.
(typearea)             This is correct!
(typearea)             Internally I'm using `paper=a4'.
(typearea)             If you'd like to set the option with \KOMAoptions,
(typearea)             you'd have to use `paper=a4' there
(typearea)             instead of `a4paper', too.
Package typearea Info: You've used standard option `oneside'.
(typearea)             This is correct!
(typearea)             Internally I'm using `twoside=false'.
(typearea)             If you'd like to set the option with \KOMAoptions,
(typearea)             you'd have to use `twoside=false' there
(typearea)             instead of `oneside', too.
\ta@hblk=\skip43
\ta@vblk=\skip44
\ta@temp=\skip45
\footheight=\skip46

Package typearea Warning: Bad type area settings!
(typearea)                The detected line width is about 22%
(typearea)                larger than the heuristically detected line width.
(typearea)                You should e.g. decrease DIV, increase fontsize
(typearea)                or change papersize.

Package typearea Info: These are the values describing the layout:
(typearea)             DIV  = 15
(typearea)             BCOR = 0.0pt
(typearea)             \paperwidth      = 597.50793pt
(typearea)              \textwidth      = 478.00635pt
(typearea)              DIV departure   = -22%
(typearea)              \evensidemargin = -12.5192pt
(typearea)              \oddsidemargin  = -12.5192pt
(typearea)             \paperheight     = 845.04694pt
(typearea)              \textheight     = 677.4003pt
(typearea)              \topmargin      = -53.33354pt
(typearea)              \headheight     = 17.0pt
(typearea)              \headsep        = 20.40001pt
(typearea)              \topskip        = 11.0pt
(typearea)              \footskip       = 47.6pt
(typearea)              \baselineskip   = 13.6pt
(typearea)              on input line 1529.
)
\c@part=\count80
\c@chapter=\count81
\c@section=\count82
\c@subsection=\count83
\c@subsubsection=\count84
\c@paragraph=\count85
\c@subparagraph=\count86
\scr@dte@part@maxnumwidth=\skip47
\scr@dte@chapter@maxnumwidth=\skip48
\scr@dte@section@maxnumwidth=\skip49
\scr@dte@subsection@maxnumwidth=\skip50
\scr@dte@subsubsection@maxnumwidth=\skip51
\scr@dte@paragraph@maxnumwidth=\skip52
\scr@dte@subparagraph@maxnumwidth=\skip53
LaTeX Info: Redefining \textsubscript on input line 4782.
\abovecaptionskip=\skip54
\belowcaptionskip=\skip55
\c@pti@nb@sid@b@x=\box27
\c@figure=\count87
\c@table=\count88
Class scrbook Info: Redefining `\numberline' on input line 5813.
\bibindent=\dimen102
)

! LaTeX Error: File `designBase.tex' not found.

Type X to quit or <RETURN> to proceed,
or enter new name. (Default extension: tex)

Enter file name: 
! Emergency stop.
<read *> 
         
l.3 \input{designBase.tex}
                          ^^M
*** (cannot \read from terminal in nonstop modes)

 
Here is how much of TeX's memory you used:
 2114 strings out of 493013
 40443 string characters out of 6136993
 192902 words of memory out of 5000000
 5728 multiletter control sequences out of 15000+600000
 3940 words of font info for 15 fonts, out of 8000000 for 9000
 1141 hyphenation exceptions out of 8191
 43i,1n,66p,8827b,137s stack positions out of 5000i,500n,10000p,200000b,80000s
!  ==> Fatal error occurred, no output PDF file produced!
