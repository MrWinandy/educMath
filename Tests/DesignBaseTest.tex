\documentclass[a4paper, 11pt, oneside, BCOR=0mm, DIV=15]{scrbook} %a4paper; landscape

\input{designBase.tex}

\begin{document}
	\chapter{firstTest}
	
	\section{SubTest}
	
	\begin{defn}
		Test
	\end{defn}
\end{document}