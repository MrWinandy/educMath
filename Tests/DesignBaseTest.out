\BOOKMARK [0][-]{chapter.1}{firstTest}{}% 1
\BOOKMARK [1][-]{section.1.1}{SubTest}{chapter.1}% 2
